engine/src/main/scala/cromwell/engine/io/gcs/GcsBatchCommandContext.scala

```diff
@@ -73,14 +74,21 @@ final case class GcsBatchCommandContext[T, U](request: GcsBatchIoCommand[T, U],
   /**
     * Queue the request for batching
     */
-  def queue(batchRequest: BatchRequest) = request.operation.queue(batchRequest, callback)
+  def queue(batchRequest: BatchRequest) = {
+    request.logIOMsgOverLimit(s"GcsBatchCommandContext.queue '$batchRequest'")
+    request.operation.queue(batchRequest, callback)
+  }
 
   /**
     * On success callback. Transform the request response to a stream-ready response that can complete the promise
     */
-  private def onSuccessCallback(response: U, httpHeaders: HttpHeaders) = handleSuccessOrNextRequest(request.onSuccess(response, httpHeaders))
+  private def onSuccessCallback(response: U, httpHeaders: HttpHeaders) = {
+    request.logIOMsgOverLimit(s"GcsBatchCommandContext.onSuccessCallback '${response.toPrettyElidedString(limit = 1000)}'")
+    handleSuccessOrNextRequest(request.onSuccess(response, httpHeaders))
+  }
 
   private def handleSuccessOrNextRequest(successResult: Either[T, GcsBatchIoCommand[T, U]]) = {
+    request.logIOMsgOverLimit(s"GcsBatchCommandContext.handleSuccessOrNextRequest '$successResult'")
```

Comment:
Seems kind of odd that we're logging the same message twice right after each other. We should probably only do that once, and use the `toPrettyElidedString` wherever we can